more/free-programming-cheatsheets.md

```diff
@@ -328,8 +328,9 @@
 * [Comprehensive Python Cheatsheet](https://gto76.github.io/python-cheatsheet) - Jure Šorn (HTML)
 * [Learn Python in Y minutes](https://learnxinyminutes.com/docs/python/) - LearnXinYMinutes (HTML)
 * [Official Matplotlib cheat sheets](https://github.com/matplotlib/cheatsheets) - Matplotlib.org (LaTeX, PDF)
-* [Python Cheat Sheet](https://websitesetup.org/python-cheat-sheet/) - WebsiteSetup (HTML, PDF, PNG)
+* [Python For Data Science Cheat Sheet](https://www.utc.fr/~jlaforet/Suppl/python-cheatsheets.pdf) - Data Camp (PDF)
```

Comment:
Looks like multiple cheatsheets were compiled into one here. Data Camp website has each page as a seperate cheatsheet. What can we do about this?

https://www.datacamp.com/cheat-sheet/category/python

Reply:
```suggestion
* [Python For Data Science Cheat Sheet](https://www.datacamp.com/cheat-sheet/python-for-data-science-a-cheat-sheet-for-beginners) - Data Camp (PDF)
```